contentcuration/contentcuration/static/js/edit_channel/vuexModules/channel.js

```diff
@@ -1,11 +1,15 @@
 const channelModule = {
   state: {
-    contentTags: []
+    contentTags: [],
+    primaryModalInUse: false,
```

Comment:
Oh, another question here - do we want to put this in the channel vuex module, or should this be in a more general module?

Reply:
I moved this stuff to a new vuex module.